Assets/Scripts/AdvancedButtons.cs

```diff
@@ -160,6 +160,29 @@ public void CreateSymbolicFolder()
             string customWIPLevelsFolder = d + Path.DirectorySeparatorChar + "Beat Saber_Data" + Path.DirectorySeparatorChar + "CustomWIPLevels";
             string dLCsFolder = d + Path.DirectorySeparatorChar + "DLC";
 
+            string pluginsFolder = d + "Plugins";
+            if (Directory.Exists(pluginsFolder)) {
+                if (File.Exists(pluginsFolder + Path.DirectorySeparatorChar + "CustomNotes.dll")) {

```

Comment:
I'm not sure if this would be better as a loop instead of a set of if statements, but it should hopefully work.